### DISCUSSION TITLE
Upgraded to 0.12 and now docker won't start

### DISCUSSION THREAD

**Discussion by silentnyte** [blakeblackshear/frigate#6150] [Ask A Question] [answered]
I have a bit of a unique setup. I am running Frigate in a "Docker Image" on TrueNAS-SCALE-22.12.2.

Everything was working fine on version 0.11. When I upgraded I am getting this error:
`Error: failed to start container "ix-chart": Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: Auto-detected mode as 'legacy' nvidia-container-cli: initialization error: nvml error: driver not loaded: unknown`

I am not passing a NVIDIA GPU but I am passing an INTEL GPU to the container so I am not sure why it is complaining. Do I have to use an NVIDIA with v0.12?

Here is my ffmpeg config
`ffmpeg:
  hwaccel_args: -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format yuv420p`

I am also confused why the stable version is reported as `stable_2304.0.1` instead of 0.12.0. I tried switching to the 0.12 tag and it also shows the same version.

**Comment by NickM-27** [comment]
what tag are you using / pulling?

**Comment by silentnyte** [comment, marked as answer]
I had some time to try and troubleshoot this and I can replicate it working and not working. If I remove my NVIDIA GT 730 from the computer I am able to start the app using "Launch Docker Image" and everything appears to be working with NO hw acceleration, only CPU. Detection, recording and integration with HA are all working. Without changing any setting and without passing any GPUs to the container and leaving detector as CPU, as soon as I plug the GT 730 back into the computer Frigate container will not start and gives this error.
`Error: failed to start container "ix-chart": Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: Auto-detected mode as 'legacy' nvidia-container-cli: initialization error: nvml error: driver not loaded: unknown`

I noted this warning when booting TrueNAS, not sure if this has anything to do with the problem.
`Jan 11 08:06:53 truenas kernel: NVRM: The NVIDIA GeForce GT 730 GPU installed in this system is
NVRM:  supported through the NVIDIA 470.xx Legacy drivers. Please
NVRM:  visit http://www.nvidia.com/object/unix.html for more
NVRM:  information.  The 515.65.01 NVIDIA driver will ignore
NVRM:  this GPU.  Continuing probe...
Jan 11 08:06:53 truenas kernel: NVRM: No NVIDIA GPU found.
Jan 11 08:06:53 truenas kernel: nvidia-nvlink: Unregistered Nvlink Core, major device number 244`

I am not sure if this is a TrueNAS issue or a Frigate issue, either way I am not sure why Frigate is attempting to use the NVIDIA card when I am not telling to.

**Reply by NickM-27** [reply]
May be due to the env vars set. You can set `NVIDIA_VISIBLE_DEVICES` to `void` and see if that helps.

**Reply by silentnyte** [reply]
Setting `NVIDIA_VISIBLE_DEVICES` to `void` solves the problem. Thank you!
